mlrun/api/api/endpoints/runtime_resources.py

```diff
@@ -221,6 +222,13 @@ def _delete_runtime_resources(
         object_id,
         mlrun.api.schemas.AuthorizationAction.delete,
     )
+
+    if not_allowed_projects_exist:
+        # some resources are not allowed, return 403
+        raise mlrun.errors.MLRunAccessDeniedError(
+            "Access denied to one or more runtime resources"
+        )
+
```

Comment:
add comments which explains why we are doing it, and TODO for when we will go to more granular permissions.